test/api/publish.test.js

```diff
@@ -266,21 +254,17 @@ describe('API versioning', () => {
     const req = getPreconfiguredRequest().accept(
       `image/webp,${ACCEPT_VALUE_MIME};version=1`
     );
-    await req
-      .send(BASIC_PAYLOAD)
-      .expect(200)
-      .expect(verifyAndDecodeJwtToken)
-      .expect(BASIC_PAYLOAD_HASH);
+
+    const res = await req.send(BASIC_PAYLOAD).expect(200);
+    verifyAndDecodeJwtToken(res);
```

Comment:
Note I call it as many times as I can, because, as it's random, I want to check characters a lot, to make sure we don't have wrong results sometimes. If we someday have an error this will be intermittent but I want it to fail as often as possible.